### PR TITLE
Added solution for HB-1

### PR DESCRIPTION
**Pull request by Nokafan**

### CODE REVIEW

File: src/main/java/com/dev/cinema/Main.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    public static void main(String[] args) {
+        Movie movie = new Movie();
+        movie.setTitle("Fast and Furious");
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
+        movieService.add(movie);
+
+        movieService.getAll().forEach(System.out::println);
+    }
+}
```
Comment (Rudiachenko) [inline src/main/java/com/dev/cinema/Main.java:18]: let's move Main class to Main/Application package

File: src/main/java/com/dev/cinema/service/impl/MovieServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import com.dev.cinema.lib.Inject;
+import com.dev.cinema.lib.Service;
+import com.dev.cinema.model.Movie;
+import com.dev.cinema.service.MovieService;
+import java.util.List;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+    @Inject
+    MovieDao movieDao;
```
Comment (Rudiachenko) [inline src/main/java/com/dev/cinema/service/impl/MovieServiceImpl.java]: access modifier?

File: src/main/java/com/dev/cinema/util/HibernateUtil.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    private static SessionFactory sessionFactory = initSessionFactory();
+
+    private HibernateUtil() {
+    }
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            throw new DataProcessingExeption("Error creating Session Factory", e);
```
Comment (Rudiachenko) [inline src/main/java/com/dev/cinema/util/HibernateUtil.java]: Here we should to throw RuntimeException

File: src/main/resources/hibernate.cfg.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/hibernate_cinema?serverTimeZone=UTC</property>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.username">admin</property>
+        <property name="connection.password">12345</property>
+        <property name="show_sql">true</property>
+        <property name="hibernate.hbm2ddl.auto">create-drop</property>
+        <mapping class="com.dev.cinema.model.Movie"/>
+    </session-factory>
+</hibernate-configuration>
```
Comment (Rudiachenko) [inline src/main/resources/hibernate.cfg.xml]: No newline at end of file

**Review by Rudiachenko — COMMENTED** [review]
Good job

File: src/main/java/com/dev/cinema/service/impl/MovieServiceImpl.java (hunk 2 above)
Comment (Nokafan) [reply src/main/java/com/dev/cinema/service/impl/MovieServiceImpl.java]: Thx, fixed.

File: src/main/java/com/dev/cinema/util/HibernateUtil.java (hunk 3 above)
Comment (Nokafan) [reply src/main/java/com/dev/cinema/util/HibernateUtil.java]: Yep, you are right. Fixed.

File: src/main/resources/hibernate.cfg.xml (hunk 4 above)
Comment (Nokafan) [reply src/main/resources/hibernate.cfg.xml]: Thx. Fixed.

**Review by Nokafan — COMMENTED** [review]
test submit

File: src/main/java/com/dev/cinema/Main.java (hunk 1 above)
Comment (Nokafan) [reply src/main/java/com/dev/cinema/Main.java:18]: No no no :) Let's leave it as is :)

**Review by dmytro-hryhoruk — APPROVED** [review]
Good job!

**Pull request merged**